cli/flox/doc/flox-edit.md

```diff
@@ -7,26 +7,55 @@ header: "Flox User Manuals"
 
 # NAME
 
-flox-edit - edit declarative format of an environment
+flox-edit - edit declarative environment configuration
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] edit [ `<options>` ]
+```
+flox [<general options>] edit
+     [-d=<path> | -r=<owner/name>]
+     [[-f=<file>] | -n=<name>]
+```
 
 # DESCRIPTION
 
-Edit environment declaratively. Has the effect of creating the
-environment if it does not exist.
+Transactionally edit the environment manifest.
```

Comment:
I think it's helpful to have here because for local environments the transactional nature is kind of the selling point i.e. what makes this more useful than just opening your editor.